docs/review.md

```diff
@@ -0,0 +1,66 @@
+# Review Guidelines
+
+The following summarises review guidelines that we follow for pull requests in
+Riot Web and other supporting repos. These are just guidelines (not strict
+rules) and may be updated over time.
+
+## Code Review
+
+When reviewing code, here are some things we look for and also things we avoid:
+
+### We review for
+
+* Correctness
+* Performance
+* Accessibility
+* Security
+* Comments and documentation where needed
+* Sharing knowledge of different areas among the team
+* Ensuring it's something we're comfortable maintaining for the long term
+
+### We should avoid
+
+* Style nits that are already handled by the linter
+* Dramatically increasing scope
+
+### Good practices
+
+* Use empathetic language
+  * See also [Mindful Communication in Code
+    Reviews](https://kickstarter.engineering/a-guide-to-mindful-communication-in-code-reviews-48aab5282e5e)
+    and [How to Do Code Reviews Like a Human](https://mtlynch.io/human-code-reviews-1/)
+* Authors should prefer smaller commits for easier reviewing and bisection
+* Reviewers should be explicit about required versus optional changes
+  * Reviews are conversations and the PR author should feel comfortable
+    discussing and pushing back on changes before making them
+* Core team should lead by example through their tone and language
+* Take the time to thank and point out good code changes
+* Using softer language like "please" and "what do you think?" goes a long way
+  towards making others feel like colleagues working towards a common goal
+
+### Workflow
+
+* Avoid force pushing to a PR after first round of review
+* Use merge commits when landing
```

Comment:
Making sure I understand the suggestion correctly; isn't this what the github merge button does? What else could you use to land a PR on `develop`?

Reply:
Yes, that's the default behaviour for GH PRs. You can change it though via the triangle in the button to instead squash or rebase. It can be confusing for people who normally work on Synapse, as their default is to squash.

So, this line of the doc is saying "use the default GH behaviour". Maybe I should expand it to clarify what not to do, though.

Reply:
I added more to this part to clarify.